src/transaction/transaction.spec.ts

```diff
@@ -1132,7 +1133,14 @@ describe('Resource population: meta', () => {
 
     const result = await externalClient.send.call({
       method: 'createBoxInNewApp',
-      args: [algorand.createTransaction.payment({ sender: testAccount, receiver: externalClient.appAddress, amount: (1).algo() })],
+      args: [
+        algorand.createTransaction.payment({
+          sender: testAccount,
+          receiver: externalClient.appAddress,
+          amount: (1).algo(),
+          signer: testAccount,
```

Comment:
This shouldn't be needed, as `fixture.newScope()` should have added the signer account.
Does this indicate some kind of behavioural change?